packages/app/features/home/screen.tsx

```diff
@@ -1,74 +1,32 @@
 import { Text, useSx, View, H1, P, Row, A } from 'dripsy'
+import { useState } from 'react'
 import { TextLink } from 'solito/link'
 import { MotiLink } from 'solito/moti'
+import SearchBar from './searchBar'
+import styles from './styles';
 
 export function HomeScreen() {
   const sx = useSx()
+  const [searchResults, setSearchResults] = useState([])
+
+  const handleSearch = async (searchText: String) => {
```

Comment:
Let's practice using the hook `useCallback` here.

Reply:
done